apps/bfd-data-npi/src/main/java/gov/cms/bfd/data/npi/lookup/NPIOrgLookup.java

```diff
@@ -15,14 +15,8 @@
 public class NPIOrgLookup {
   private static final Logger LOGGER = LoggerFactory.getLogger(NPIOrgLookup.class);
 
-  /** A fake npi number used for testing. */
-  public static final String FAKE_NPI_NUMBER = "0000000000";
-
-  /** A fake org name display that is associated with the FAKE_NPI_ORG_NAME. */
-  public static final String FAKE_NPI_ORG_NAME = "Fake ORG Name";
-
   /** Hashmap to keep the org names. */
-  private Map<String, String> npiOrgHashMap = new HashMap<>();
+  public Map<String, String> npiOrgHashMap = new HashMap<>();
```

Comment:
Same comment as above, this can remain private with a separate constructor that accepts this